darts/metrics/metrics.py

```diff
@@ -1212,3 +1212,81 @@ def rho_risk(
 
     rho_loss = 2 * (z_true - z_hat_rho) * (rho * pred_below - (1 - rho) * pred_above)
     return rho_loss / z_true
+
+
+# rho-risk (quantile risk)
```

Comment:
this comment should be removed, probably a leftover from the rho_risk copy-paste

Reply:
Ahhh, you caught me red-handed I was absolutely certain I hid all the evidence of my little copy and paste shenanigan ;P